doc/guide/feature-machines.xml

```diff
@@ -19,7 +19,7 @@
     <filename>/etc/ssh/ssh_known_hosts</filename>.</para>
 
   <para>The machine data is stored in
-    <filename>/etc/cockpit/machines.d/*.json</filename>.  Settings in
+    <filename>${XDG_CONFIG_DIRS:-/etc}/cockpit/machines.d/*.json</filename>.  Settings in
```

Comment:
I feel like this is a bit too advanced-shell-syntax for documentation...  Why not just spell out explicitly the steps we take to consider what we consider the cockpit config directory to be?